### ISSUE TITLE
vault-plugin-secrets-gcp not working with Vault 0.9.6

### ISSUE THREAD

**Issue by peric** [hashicorp/vault-plugin-secrets-gcp#40]
In the documentation (https://github.com/hashicorp/vault-plugin-secrets-gcp#usage) it states that: 

> If you are using Vault v0.9.6 or above, this plugin is packaged with Vault and by default can be enabled by running

Although, I don't find this to be true, or I am missing something.

I have tested this both on our production environment and locally in Docker container where I use `vault:0.9.6` image (https://hub.docker.com/_/vault/). Both environments are definitely running on Vault 0.9.6.

When I try to run `vault secrets enable gcp`, I receive:

```
URL: POST http://vault:8200/v1/sys/mounts/gcp
Code: 400. Errors:

* unknown backend type: gcp
```

Also, when I try to run `vault secrets list`, I receive a list of 5 paths without `gcp` being listed.

Any ideas?

**Comment by emilymye** [comment]
Ah, documentation error. The GCP secrets backend was available in 0.10.0 (the auth backend was 0.9.6). Also, please do try to stay somewhat closer to newer releases of Vault if possible! A lot of improvements, including fixes to bugs and potentially significant security patches, have been released in newer versions of both Vault core and this plugin.

**Comment by peric** [comment]
Thanks for your answer. We are moving away from Vault 0.9.6 soon, I hope. Until then, I'll probably use a custom build.
